Feature_Guide/Data_Pipelines/README.md

```diff
@@ -0,0 +1,437 @@
+    ]
+
+    # Query
+    query_response = client.infer(model_name="ensemble_model",
+                                  inputs=input_tensors,
+                                  outputs=output)
+
+    print(query_response.as_numpy("ensemble_output_string"))
+    print(query_response.as_numpy("ensemble_output_UINT8_array"))
+    print(query_response.as_numpy("ensemble_output_INT8_array"))
+    print(query_response.as_numpy("ensemble_output_FP32_image"))
+    print(query_response.as_numpy("ensemble_output_bool"))
+```
+
+There are two moving parts in the client code that need to be understood. First, let's look at seting the input and output.
+```
+# Input
+input_tensors = [
+    httpclient.InferInput("ensemble_input_string", text_obj.shape,np_to_triton_dtype(text_obj.dtype)),
+    httpclient.InferInput("ensemble_input_UINT8_array", uint8_array.shape, datatype="UINT8"),
+    httpclient.InferInput("ensemble_input_INT8_array", int8_array.shape, datatype="INT8"),
+    httpclient.InferInput("ensemble_input_FP32_image", image.shape, datatype="FP32"),
+    httpclient.InferInput("ensemble_input_bool", boolean.shape, datatype="BOOL")
+]
+input_tensors[0].set_data_from_numpy(text_obj)
+input_tensors[1].set_data_from_numpy(uint8_array)
+input_tensors[2].set_data_from_numpy(int8_array)
+input_tensors[3].set_data_from_numpy(image)
+input_tensors[4].set_data_from_numpy(boolean)
+
+# Output
+output = [
+    httpclient.InferRequestedOutput("ensemble_output_string"),
+    httpclient.InferRequestedOutput("ensemble_output_UINT8_array"),
+    httpclient.InferRequestedOutput("ensemble_output_INT8_array"),
+    httpclient.InferRequestedOutput("ensemble_output_FP32_image"),
+    httpclient.InferRequestedOutput("ensemble_output_bool")
+]
+
+```
+In this case we are using the `http` client, and specifying the names of the input and outputs along with the expected datatype. Notice that in this case we are using the ensemble inputs/outputs, for instance `ensemble_input_string` for the string input to the ensemble. If you want to just query a singluar model, you can switch these for the layers for the model and fire the query to the specific model in the `client.infer(...)` function. 
+
+```
+# Creating a client for the server
+client = httpclient.InferenceServerClient(url="localhost:8000")
+
+# Querying the Server
+query_response = client.infer(model_name="ensemble_model",
+                                inputs=input_tensors,
+                                outputs=output)
+
+print(query_response.as_numpy("ensemble_output_string"))
+print(query_response.as_numpy("ensemble_output_UINT8_array"))
+print(query_response.as_numpy("ensemble_output_INT8_array"))
+print(query_response.as_numpy("ensemble_output_FP32_image"))
+print(query_response.as_numpy("ensemble_output_bool"))
+```
+
+The second part to look at is the general client API. This is the same as any Triton Client, simply use the python client API to query the server.
```

Comment:
This line doesn't seem like its telling me anything. Can you re-word or elaborate on it?